windows/src/engine/keyman/UfrmKeyman7Main.pas

```diff
@@ -225,6 +228,9 @@   TfrmKeyman7Main = class(TForm)
     FHotkeyWindow: HWND;
     FHotkeys: TIntegerList;
 
+    FInputPane: TFrameworkInputPane;
```

Comment:
Any thoughts on mkaing

Reply:
We could do, but because the update logic is in `TfrmKeyman7Main`, I think the variable probably belongs there as well :O)